tests/drum/test_perf_and_validation.py

```diff
@@ -0,0 +1,104 @@
+import pytest
```

Comment:
I feel like this could be split into a perf file and a validation file.

Reply:
splitted